### ISSUE TITLE
inputs.md

### ISSUE THREAD

**Issue by nicholebarry** [EoRImaging/FHD#186]
We should make an inputs.md which describes all of the input catalogs and calibration files. It's too hard to parse what exactly went into each file from just the file name alone.

**Comment by isullivan** [comment]
This is a great idea! It would not be difficult to add a simple function in `fhd_save_io.pro` to do this for parameters that are restored from previous runs, and not too much work to call that function everywhere else where something is read from disk.
